modules/persistent-storage-csi-vsphere-rwx.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+[id="persistent-storage-csi-vsphere-rwx_{context}"]
+= ReadWriteMany vSphere volume support
+
+If the underlying vSphere environment supports the vSAN file service, then vSphere Container Storage Interface (CSI) Driver Operator installed by
```

Comment:
How can a user know if the the _vSphere environment_ supports **vSAN file service** ? 
Outside of 'trying to create a volume' what tells a user that this is 'enabled'? Or would function; and not result in a pod failing to deploy (or provide the correct storage).

https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vsan.doc/GUID-CA9CF043-9434-454E-86E7-DCA9AD9B0C09.html seems to answer this: 

1. Navigate to the vSAN cluster and click Configure > vSAN > Services.
2. On the File Service row (check the Enabled Setting)?